actions:
  goto_r1:
    attr:
      pose:
      - - 2.5
        - 2.5
        - 0
      - - 0
        - 0
        - 0
        - 1
      region: r1
    type: move
    weight: 10
  goto_r10:
    attr:
      pose:
      - - 22.5
        - 7.5
        - 0
      - - 0
        - 0
        - 0
        - 1
      region: r10
    type: move
    weight: 10
  goto_r11:
    attr:
      pose:
      - - 2.5
        - 12.5
        - 0
      - - 0
        - 0
        - 0
        - 1
      region: r11
    type: move
    weight: 10
  goto_r12:
    attr:
      pose:
      - - 7.5
        - 12.5
        - 0
      - - 0
        - 0
        - 0
        - 1
      region: r12
    type: move
    weight: 10
  goto_r13:
    attr:
      pose:
      - - 12.5
        - 12.5
        - 0
      - - 0
        - 0
        - 0
        - 1
      region: r13
    type: move
    weight: 10
  goto_r14:
    attr:
      pose:
      - - 17.5
        - 12.5
        - 0
      - - 0
        - 0
        - 0
        - 1
      region: r14
    type: move
    weight: 10
  goto_r15:
    attr:
      pose:
      - - 22.5
        - 12.5
        - 0
      - - 0
        - 0
        - 0
        - 1
      region: r15
    type: move
    weight: 10
  goto_r16:
    attr:
      pose:
      - - 2.5
        - 17.5
        - 0
      - - 0
        - 0
        - 0
        - 1
      region: r16
    type: move
    weight: 10
  goto_r17:
    attr:
      pose:
      - - 7.5
        - 17.5
        - 0
      - - 0
        - 0
        - 0
        - 1
      region: r17
    type: move
    weight: 10
  goto_r18:
    attr:
      pose:
      - - 12.5
        - 17.5
        - 0
      - - 0
        - 0
        - 0
        - 1
      region: r18
    type: move
    weight: 10
  goto_r19:
    attr:
      pose:
      - - 17.5
        - 17.5
        - 0
      - - 0
        - 0
        - 0
        - 1
      region: r19
    type: move
    weight: 10
  goto_r2:
    attr:
      pose:
      - - 7.5
        - 2.5
        - 0
      - - 0
        - 0
        - 0
        - 1
      region: r2
    type: move
    weight: 10
  goto_r20:
    attr:
      pose:
      - - 22.5
        - 17.5
        - 0
      - - 0
        - 0
        - 0
        - 1
      region: r20
    type: move
    weight: 10
  goto_r21:
    attr:
      pose:
      - - 2.5
        - 22.5
        - 0
      - - 0
        - 0
        - 0
        - 1
      region: r21
    type: move
    weight: 10
  goto_r22:
    attr:
      pose:
      - - 7.5
        - 22.5
        - 0
      - - 0
        - 0
        - 0
        - 1
      region: r22
    type: move
    weight: 10
  goto_r23:
    attr:
      pose:
      - - 12.5
        - 22.5
        - 0
      - - 0
        - 0
        - 0
        - 1
      region: r23
    type: move
    weight: 10
  goto_r24:
    attr:
      pose:
      - - 17.5
        - 22.5
        - 0
      - - 0
        - 0
        - 0
        - 1
      region: r24
    type: move
    weight: 10
  goto_r25:
    attr:
      pose:
      - - 22.5
        - 22.5
        - 0
      - - 0
        - 0
        - 0
        - 1
      region: r25
    type: move
    weight: 10
  goto_r3:
    attr:
      pose:
      - - 12.5
        - 2.5
        - 0
      - - 0
        - 0
        - 0
        - 1
      region: r3
    type: move
    weight: 10
  goto_r4:
    attr:
      pose:
      - - 17.5
        - 2.5
        - 0
      - - 0
        - 0
        - 0
        - 1
      region: r4
    type: move
    weight: 10
  goto_r5:
    attr:
      pose:
      - - 22.5
        - 2.5
        - 0
      - - 0
        - 0
        - 0
        - 1
      region: r5
    type: move
    weight: 10
  goto_r6:
    attr:
      pose:
      - - 2.5
        - 7.5
        - 0
      - - 0
        - 0
        - 0
        - 1
      region: r6
    type: move
    weight: 10
  goto_r7:
    attr:
      pose:
      - - 7.5
        - 7.5
        - 0
      - - 0
        - 0
        - 0
        - 1
      region: r7
    type: move
    weight: 10
  goto_r8:
    attr:
      pose:
      - - 12.5
        - 7.5
        - 0
      - - 0
        - 0
        - 0
        - 1
      region: r8
    type: move
    weight: 10
  goto_r9:
    attr:
      pose:
      - - 17.5
        - 7.5
        - 0
      - - 0
        - 0
        - 0
        - 1
      region: r9
    type: move
    weight: 10
  goto_s0:
    attr:
      pose:
      - - 3.0
        - 3.0
        - 0
      - - !!python/object/apply:numpy.core.multiarray.scalar
          - &id001 !!python/object/apply:numpy.dtype
            args:
            - f8
            - 0
            - 1
            state: !!python/tuple
            - 3
            - <
            - null
            - null
            - null
            - -1
            - -1
            - 0
          - !!binary |
            AAAAAAAAAAA=
        - !!python/object/apply:numpy.core.multiarray.scalar
          - *id001
          - !!binary |
            AAAAAAAAAAA=
        - !!python/object/apply:numpy.core.multiarray.scalar
          - *id001
          - !!binary |
            AAAAAAAAAAA=
        - !!python/object/apply:numpy.core.multiarray.scalar
          - *id001
          - !!binary |
            AAAAAAAA8D8=
      region: s0
    guard: '1'
    type: move
    weight: 10
  goto_s1:
    attr:
      pose:
      - - 5.0
        - 5.0
        - 0
      - - !!python/object/apply:numpy.core.multiarray.scalar
          - *id001
          - !!binary |
            AAAAAAAAAAA=
        - !!python/object/apply:numpy.core.multiarray.scalar
          - *id001
          - !!binary |
            AAAAAAAAAAA=
        - !!python/object/apply:numpy.core.multiarray.scalar
          - *id001
          - !!binary |
            AAAAAAAAAAA=
        - !!python/object/apply:numpy.core.multiarray.scalar
          - *id001
          - !!binary |
            AAAAAAAA8D8=
      region: s1
    guard: '1'
    type: move
    weight: 10
state_dim:
- 2d_pose_region
state_models:
  2d_pose_region:
    initial: ''
    nodes:
      r1:
        attr:
          hysteresis: 1.0
          length: 5.0
          pose:
          - - 2.5
            - 2.5
          - - 0
          type: square
        connected_to:
          r1: goto_r1
          r2: goto_r2
          r6: goto_r6
          s0: goto_s0
      r10:
        attr:
          hysteresis: 1.0
          length: 5.0
          pose:
          - - 22.5
            - 7.5
          - - 0
          type: square
        connected_to:
          r10: goto_r10
          r15: goto_r15
          r5: goto_r5
          r9: goto_r9
      r11:
        attr:
          hysteresis: 1.0
          length: 5.0
          pose:
          - - 2.5
            - 12.5
          - - 0
          type: square
        connected_to:
          r11: goto_r11
          r12: goto_r12
          r16: goto_r16
          r6: goto_r6
      r12:
        attr:
          hysteresis: 1.0
          length: 5.0
          pose:
          - - 7.5
            - 12.5
          - - 0
          type: square
        connected_to:
          r11: goto_r11
          r12: goto_r12
          r13: goto_r13
          r17: goto_r17
          r7: goto_r7
      r13:
        attr:
          hysteresis: 1.0
          length: 5.0
          pose:
          - - 12.5
            - 12.5
          - - 0
          type: square
        connected_to:
          r12: goto_r12
          r13: goto_r13
          r14: goto_r14
          r18: goto_r18
          r8: goto_r8
      r14:
        attr:
          hysteresis: 1.0
          length: 5.0
          pose:
          - - 17.5
            - 12.5
          - - 0
          type: square
        connected_to:
          r13: goto_r13
          r14: goto_r14
          r15: goto_r15
          r19: goto_r19
          r9: goto_r9
      r15:
        attr:
          hysteresis: 1.0
          length: 5.0
          pose:
          - - 22.5
            - 12.5
          - - 0
          type: square
        connected_to:
          r10: goto_r10
          r14: goto_r14
          r15: goto_r15
          r20: goto_r20
      r16:
        attr:
          hysteresis: 1.0
          length: 5.0
          pose:
          - - 2.5
            - 17.5
          - - 0
          type: square
        connected_to:
          r11: goto_r11
          r16: goto_r16
          r17: goto_r17
          r21: goto_r21
      r17:
        attr:
          hysteresis: 1.0
          length: 5.0
          pose:
          - - 7.5
            - 17.5
          - - 0
          type: square
        connected_to:
          r12: goto_r12
          r16: goto_r16
          r17: goto_r17
          r18: goto_r18
          r22: goto_r22
      r18:
        attr:
          hysteresis: 1.0
          length: 5.0
          pose:
          - - 12.5
            - 17.5
          - - 0
          type: square
        connected_to:
          r13: goto_r13
          r17: goto_r17
          r18: goto_r18
          r19: goto_r19
          r23: goto_r23
      r19:
        attr:
          hysteresis: 1.0
          length: 5.0
          pose:
          - - 17.5
            - 17.5
          - - 0
          type: square
        connected_to:
          r14: goto_r14
          r18: goto_r18
          r19: goto_r19
          r20: goto_r20
          r24: goto_r24
      r2:
        attr:
          hysteresis: 1.0
          length: 5.0
          pose:
          - - 7.5
            - 2.5
          - - 0
          type: square
        connected_to:
          r1: goto_r1
          r2: goto_r2
          r3: goto_r3
          r7: goto_r7
          s0: goto_s0
      r20:
        attr:
          hysteresis: 1.0
          length: 5.0
          pose:
          - - 22.5
            - 17.5
          - - 0
          type: square
        connected_to:
          r15: goto_r15
          r19: goto_r19
          r20: goto_r20
          r25: goto_r25
      r21:
        attr:
          hysteresis: 1.0
          length: 5.0
          pose:
          - - 2.5
            - 22.5
          - - 0
          type: square
        connected_to:
          r16: goto_r16
          r21: goto_r21
          r22: goto_r22
      r22:
        attr:
          hysteresis: 1.0
          length: 5.0
          pose:
          - - 7.5
            - 22.5
          - - 0
          type: square
        connected_to:
          r17: goto_r17
          r21: goto_r21
          r22: goto_r22
          r23: goto_r23
      r23:
        attr:
          hysteresis: 1.0
          length: 5.0
          pose:
          - - 12.5
            - 22.5
          - - 0
          type: square
        connected_to:
          r18: goto_r18
          r22: goto_r22
          r23: goto_r23
          r24: goto_r24
      r24:
        attr:
          hysteresis: 1.0
          length: 5.0
          pose:
          - - 17.5
            - 22.5
          - - 0
          type: square
        connected_to:
          r19: goto_r19
          r23: goto_r23
          r24: goto_r24
          r25: goto_r25
      r25:
        attr:
          hysteresis: 1.0
          length: 5.0
          pose:
          - - 22.5
            - 22.5
          - - 0
          type: square
        connected_to:
          r20: goto_r20
          r24: goto_r24
          r25: goto_r25
      r3:
        attr:
          hysteresis: 1.0
          length: 5.0
          pose:
          - - 12.5
            - 2.5
          - - 0
          type: square
        connected_to:
          r2: goto_r2
          r3: goto_r3
          r4: goto_r4
          r8: goto_r8
      r4:
        attr:
          hysteresis: 1.0
          length: 5.0
          pose:
          - - 17.5
            - 2.5
          - - 0
          type: square
        connected_to:
          r3: goto_r3
          r4: goto_r4
          r5: goto_r5
          r9: goto_r9
      r5:
        attr:
          hysteresis: 1.0
          length: 5.0
          pose:
          - - 22.5
            - 2.5
          - - 0
          type: square
        connected_to:
          r10: goto_r10
          r4: goto_r4
          r5: goto_r5
      r6:
        attr:
          hysteresis: 1.0
          length: 5.0
          pose:
          - - 2.5
            - 7.5
          - - 0
          type: square
        connected_to:
          r1: goto_r1
          r11: goto_r11
          r6: goto_r6
          r7: goto_r7
          s0: goto_s0
          s1: goto_s1
      r7:
        attr:
          hysteresis: 1.0
          length: 5.0
          pose:
          - - 7.5
            - 7.5
          - - 0
          type: square
        connected_to:
          r12: goto_r12
          r2: goto_r2
          r6: goto_r6
          r7: goto_r7
          r8: goto_r8
      r8:
        attr:
          hysteresis: 1.0
          length: 5.0
          pose:
          - - 12.5
            - 7.5
          - - 0
          type: square
        connected_to:
          r13: goto_r13
          r3: goto_r3
          r7: goto_r7
          r8: goto_r8
          r9: goto_r9
      r9:
        attr:
          hysteresis: 1.0
          length: 5.0
          pose:
          - - 17.5
            - 7.5
          - - 0
          type: square
        connected_to:
          r10: goto_r10
          r14: goto_r14
          r4: goto_r4
          r8: goto_r8
          r9: goto_r9
      s0:
        attr:
          angle_hysteresis: 0.1
          angle_threshold: 0.2
          dist_hysteresis: 1.0
          pose:
          - - 3.0
            - 3.0
          - - 0.0
          radius: 2.0
          type: station
        connected_to:
          r1: goto_r1
          r2: goto_r2
          r6: goto_r6
          s0: goto_s0
      s1:
        attr:
          angle_hysteresis: 0.1
          angle_threshold: 0.2
          dist_hysteresis: 1.0
          pose:
          - - 5.0
            - 5.0
          - - 0.0
          radius: 2.0
          type: station
        connected_to:
          r6: goto_r6
          s1: goto_s1
    ts_type: 2d_pose_region
